src/main/kotlin/no/nav/syfo/infrastructure/cronjob/PubliserVurderingerCronjob.kt

```diff
@@ -0,0 +1,28 @@
+package no.nav.syfo.infrastructure.cronjob
+
+import net.logstash.logback.argument.StructuredArguments
+import no.nav.syfo.application.service.VurderingService
+import org.slf4j.LoggerFactory
+
+class PubliserVurderingerCronjob(
+    private val vurderingService: VurderingService,
+) : Cronjob {
+    override val initialDelayMinutes: Long = 2
+    override val intervalDelayMinutes: Long = 1
```

Comment:
Trenger den å gå så ofte egentlig? For ispersonoppgave sin del vil vel det bare være krise hvis man setter oppfylt rett før man sender ut oppgave om utgått varsel? Men hvis vi er redd for at det skjer ofte så kan vi kjøre på med 1 minutt 🤷🏼‍♂️

Reply:
Ja, det var vel en tanke om at oversikten skulle oppdateres relativt raskt.